src/components/Data/Documents/Page.vue

```diff
@@ -423,8 +423,13 @@ export default {
     },
     // LIST (FETCH & SEARCH)
     // =========================================================================
-    loadAllTheThings() {
-      this.loadMappingInfo()
+    async loadAllTheThings() {
+      this.loading = true
+      try {
```

Comment:
using a try...catch block is a nice idea! I think you can put all the calls of the function inside it and display a toast rather than simply stop the loader